### ISSUE TITLE
CSN model training does not converge

### ISSUE THREAD

**Issue by gfjiangly** [open-mmlab/mmaction#175]
Thanks for providing this comprehensive code base.
When I tried to train the newly updated model "CSN", loss has been maintained at around 0.7, although a dozen epochs have been trained.

**Comment by dagongji10** [comment]
@gfjiangly Do you know what is CSN? I try to search paper of CSN but failed, could you tell me CSN's full name? What kind of network does it represent?

**Comment by kennymckormick** [comment]
We haven't released training config for CSN yet, you can refer to the original paper for the training setting. You can also test the trained model to see if accuracy is OK.
